README.md

```diff
@@ -146,6 +146,7 @@ checkoutservice --> currencyservice
 checkoutservice --> emailservice
 checkoutservice --> paymentservice
 checkoutservice --> shippingservice
+checkoutservice --> |evalFlag| featureflagfeservice
```

Comment:
In regard to naming the arrows: do we think we'll in the future call out all the request types to and from services?

Reply:
Yeah I think so. Maybe not necessarily in place. We can have multiple versions with various levels of detail. 

I'd like to add in OTel components too